build.sh

```
@@ -0,0 +1,372 @@
+    
+    # Copy LICENSE if exists
+    if [ -f "LICENSE" ]; then
+        cp "LICENSE" "$DIST_DIR/$PRODUCT_FOLDER/"
+    fi
+    
+    # Create zip file
+    echo "Creating zip archive..."
+    (cd "$DIST_DIR" && zip -r "$PRODUCT_FOLDER.zip" "$PRODUCT_FOLDER")
+    
+    # Clean up unzipped folder
+    rm -rf "$DIST_DIR/$PRODUCT_FOLDER"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)